thenewboston_node/business_logic/models/base.py

```diff
@@ -43,6 +43,41 @@
 
 UNCOMPACT_KEY_MAP = {value: key for key, value in COMPACT_KEY_MAP.items()}
 
+
+def hex_to_bytes(value):
+    if value is None:
+        return value
+    return bytes.fromhex(value)
+
+
+def bytes_to_hex(value):
+    if value is None:
+        return value
+    return value.hex()
+
+
+COMPACT_VALUE_MAP = {
```

Comment:
also we use hexadecimals as keys here:
```
    accounts: dict[str, AccountBalance]
```

and here:

```
    updated_balances: dict[str, BlockAccountBalance] = {}

```

Please, take care of it too

Reply:
Done